test/powershell/Modules/Microsoft.PowerShell.Management/Set-Location.Tests.ps1

```diff
@@ -213,4 +213,56 @@ Describe "Set-Location" -Tags "CI" {
             $PWD.Path | Should -Be $location.Path
         }
     }
+
+    Context 'Test Set-Location updates UNIX PWD environment variable' {
+        It 'Should set the PWD environment variable to the new filesystem location' -Skip:$IsWindows {
+            # Use $env:TMPDIR or /tmp to ensure that we're testing a FileSystem path
+            if ($env:TMPDIR) {
+                $TempDirectory = $env:TMPDIR
+            } else {
+                $TempDirectory = '/tmp'
+            }
+            $path = 'Directory1'
+            $location = Join-Path $TempDirectory $path
```

Comment:
Another rule from best practice, use named parameters instead of positional.

eg: `Join-Path $TempDirectory $path` --> `Join-Path -Path $TempDirectory -ChildPath $Path`